examples/features/JumpTable.huff

```diff
@@ -0,0 +1,32 @@
+#define macro MAIN() = takes(0) returns(0) {
+    0x21 calldataload                // [y]
+    0x01 calldataload                // [y, x]
+    2                                // [y, x, 2]
+    0x0 calldataload 0x0 byte        // [y, x, 2, b]
+    0x1 shl                          // [y, x, 2, b * 2]
```

Comment:
that you have to shl indicates the table isn't really packed

Reply:
are jumpdest labels always two bytes in huff?

Reply:
For now yes, planning on supporting arbitrary sizes in the future. The non packed variant pads labels to 32 bytes.